modules/apps/captcha/captcha-impl/src/main/java/com/liferay/captcha/internal/provider/CaptchaProviderImpl.java

```diff
@@ -30,8 +28,8 @@
 public class CaptchaProviderImpl implements CaptchaProvider {
 
 	@Override
-	public Captcha getCaptcha() {
-		String captchaClassName = _captchaConfiguration.captchaEngine();
+	public Captcha getCaptcha(CaptchaConfiguration captchaConfiguration) {
```

Comment:
Hey @ChrisKian, I think we can remove the `configurationPid = "com.liferay.captcha.configuration.CaptchaConfiguration"` from the Component annotation since we are using the `ConfigutationProvider` api to retrieve the `CaptchaConfiguration` now.